### DISCUSSION TITLE
Idk what to do in the geany folder

### DISCUSSION THREAD

**Discussion by AnasAyadKamal** [raysan5/raylib#1933] [Q&A]
idk what should i do with these files at all, can someone tell me wat to do????[this one](https://github.com/raysan5/raylib/tree/master/projects/Geany)
im using linux/gnu (peppermint 10 ) btw

**Comment by chriscamacho** [comment]
I've never bothered with them to be honest, geany is bright enough that if you have raylib.h (and usually raymath.h) loaded in other tabs, then autocomplete just works...

obviously the bat files are windows specific, you can get geany to run make for you, but to be honest I usually just have a separate terminal window open as I find it more convenient

Its worth having a look at https://www.geany.org/manual/current/index.html

**Reply by AnasAyadKamal** [reply]
huh? mate thanks for the answer i appreciate it but i just wanna know what to do wit these files

**Reply by AnasAyadKamal** [reply]
and it doesnt find the raylib.h file bruh

**Comment by chriscamacho** [comment]
the manual I linked to tells you ! basically the tags file goes in the one of the configuration paths, the .bat files are not pertinent to Linux 

just load raylib.h into geany, and autocomplete will work, when you are editing a file in another tab

**Reply by AnasAyadKamal** [reply]
i did that but it doesnt
